redis_enterprise/datadog_checks/redis_enterprise/redis_enterprise.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any  # noqa: F401
+
+from datadog_checks.base import AgentCheck, ConfigurationError, OpenMetricsBaseCheckV2  # noqa: F401
+
+
+class RedisEnterpriseCheck(OpenMetricsBaseCheckV2):
+
+    # This will be the prefix of every metric and service check the integration sends
+    # __NAMESPACE__ = 'redis_enterprise'
```

Comment:
```suggestion
    __NAMESPACE__ = 'redis_enterprise'
```

Reply:
How does this affect the namespace declared in the yaml file, should they be the same? are they both applied?
    ## @param namespace - the namespace used by the metrics - required
    ## this value is used by the various dashboards provided; change it and it must be replaced in the dashboard's .json
    namespace: "rdse"

Reply:
The question is moot, since we no longer expose a config field `namespace`. You should use this field and set it to what you expect in the dashboards, eg `rdse`.

Reply:
@j8-redis bumping the above!